<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDV</class>
 <widget class="QDialog" name="RDV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>722</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>722</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>722</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MED++</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter RDV</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	background-image: url(:/AjoutRDV.png);

}
</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>410</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="retour">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>400</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>400</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_etat">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>460</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>572</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier RDV</string>
    </attribute>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_6
{
	background-image: url(:/ModifierRDV.png);
}
</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="retour_2">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>410</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>664</x>
        <y>572</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>410</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_etat_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>460</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer RDV</string>
    </attribute>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>691</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_7
{
	background-image: url(:/SuppRDV.png);
}
</string>
     </property>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>650</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Delete">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-20</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>380</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>500</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Imprint MT Shadow</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Raison de suppression:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>440</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>630</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer les datas</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>370</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Afficher RDV</string>
    </attribute>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_4
{
 
	background-image: url(:/AffichRDV.png);
}</string>
     </property>
     <widget class="QTableView" name="tablerdv">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>631</width>
        <height>311</height>
       </rect>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>120</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner la date:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tablerdv_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>580</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
